src/redux/utils.js

```diff
@@ -1,9 +1,9 @@
-import { createAsyncThunk } from '@reduxjs/toolkit';
+import { fetchBooks } from '@reduxjs/toolkit';
```

Comment:
- [ ] Excellent work on implementing the` createAsyncThunk` for your `getBooks` action. However, I think you misunderstood what the previous reviewer meant. Because currently, you have removed `creareAsyncThunk`which as specified in the project requirements needs to be implemented. So please ensure to reuse `createAsyncThunk` for all asynchronous actions as specified :+1: 
- [ ] As a result of this, your app is currently broken so please fix this :+1: 
- Reference
![image](https://github.com/aima98/Bookstore/assets/102699958/eac41078-93dc-4e62-8e72-0fe0e93a8cce)

- [ ] All you had to do earlier was to update the action name from `getBooks` to `fetchBooks` to maintain consistency with the naming conventions used throughout the project. So please implement this. Keep up the good work! :100: 
- [ ] You can change your code back to what it was before and just change the action name like i mentioned earlier :wink: